napari_allencell_annotator/_tests/view/viewer_test.py

```diff
@@ -0,0 +1,113 @@
+from pathlib import Path
+
+import napari
+from bioio import BioImage
+
+import napari_allencell_annotator
+from napari.layers import Points
+
+from napari_allencell_annotator.util.image_utils import ImageUtils
+from napari_allencell_annotator.view.viewer import Viewer, PointsLayerMode
+import numpy as np
+import pytest
+from napari.utils import Colormap
+
+
+@pytest.fixture
+def viewer(make_napari_viewer: napari.Viewer) -> Viewer:
+    return Viewer(make_napari_viewer())
+
+
+def test_all_points_layer(viewer: Viewer) -> None:
+    # ARRANGE
+    viewer.viewer.add_shapes()
+    test_points_layer1: Points = viewer.create_points_layer("test1", "blue", True, 2)
+    test_points_layer2: Points = viewer.create_points_layer("test2", "blue", True, 2)
+
+    # ACT
+    all_points_layer: list[Points] = viewer.get_all_points_layers()
+
+    # ASSERT
+    assert len(all_points_layer) == 2
+    assert test_points_layer1 in all_points_layer
+    assert test_points_layer2 in all_points_layer
+
+
+def test_add_image(viewer: Viewer) -> None:
+    # ARRANGE
+    test_path: Path = (
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "image_types" / "img.ome.tiff"
+    )
+    test_image: BioImage = ImageUtils(test_path).get_image()
+
+    # ACT
+    viewer.add_image(test_image)
+
+    # ASSERT
+    assert len(viewer.get_layers()) == 1
+    np.testing.assert_array_equal(viewer.get_layers()[0].data.compute(), test_image.get_dask_stack().compute())
+
+
+def test_create_points_layer(viewer: Viewer) -> None:
+    # ACT
+    test_points_layer: Points = viewer.create_points_layer("test", "blue", True, 2)
+
+    # ASSERT
+    assert test_points_layer in viewer.get_all_points_layers()
+    assert len(viewer.get_all_points_layers()) == 1
+    assert test_points_layer.name == "test"
+    assert test_points_layer.visible
+    assert test_points_layer.ndim == 2
```

Comment:
remove dimension from `create_points_layer()`

Reply:
removed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/91b886b85e3754b6e4ed509d04bf4334c7819c19

Reply:
I removed `ndim` from `create_points_layer()` but I still have to pass ndim=6 when I call `self.viewer.add_points()`. Otherwise, point coordinates would be saved as 2D.

Reply:
discussed- get dims from viewer

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/0e863856e340a7c5a741f3669ad9db62993beb65